float64[3] r_star
float64[3] v_star
float64[3] EEposition
float64[42] jacobian
float64[21] J_translation
float64[21] J_translation_pinv
float64[42] jacobian_array
float64[7] q_c
